components/calendar/src/hebrew.rs

```diff
@@ -218,15 +240,22 @@ impl Calendar for Hebrew {
         Self::year_as_hebrew(date.0.year)
     }
 
-    fn month(&self, date: &Self::DateInner) -> types::FormattableMonth {
+    fn month(&self, date: &Self::DateInner) -> FormattableMonth {
         let mut ordinal = date.0.month;
         let is_leap_year = Self::is_leap_year(date.0.year);
 
-        if is_leap_year && ordinal == 6 {
-            return types::FormattableMonth {
-                ordinal: ordinal as u32,
-                code: types::MonthCode(tinystr!(4, "M05L")),
-            };
+        if is_leap_year {
+            if ordinal == 6 {
+                return types::FormattableMonth {
+                    ordinal: ordinal as u32,
+                    code: types::MonthCode(tinystr!(4, "M05L")),
+                };
+            } else if ordinal == 7 {
+                return types::FormattableMonth {
+                    ordinal: ordinal as u32,
+                    code: types::MonthCode(tinystr!(4, "M06L")),
```

Comment:
Thought: I thought this was already working but I guess not! Maybe we did it in the formatting code but not in the calendar code